src/analysis/normed_space/bounded_linear_maps.lean

```diff
@@ -228,6 +228,34 @@ end
 
 section bilinear_map
 
+namespace continuous_linear_map
+
+lemma map_add_left (f : E →L[𝕜] F →L[𝕜] G) {x x' : E} {y : F} : f (x + x') y = f x y + f x' y :=
+by rw [f.map_add, add_apply]
```

Comment:
For `linear_map` we call this `map_add\_2`

Reply:
I'll try to follow the `linear_map` conventions.

Reply:
Either that or change the `linear_map` ones. In theory you should be able to state these with `add_monoid_hom_class` so that we only need them once, but I don't know if that works in practice.

Reply:
The `linear_map` API works well.